WordPress/gutenberg
# VStack: Convert component to TypeScript

## What?
Converts the `VStack` component to TypeScript.

## Why?
Part of the @wordpress/components's TypeScript migration (https://github.com/WordPress/gutenberg/issues/35744).

## How?
- Refactors the current `VStack` to TypeScript.

## Testing Instructions
1. Run npm run dev and ensure no typing errors
3. Test the various v-stack within the block/site editors in Gutenberg still work
